knowledge_runtime/knowledge_runtime/config.py

```
@@ -0,0 +1,52 @@
+
+class Settings(BaseSettings):
+    """Runtime service settings loaded from environment variables."""
+
+    model_config = SettingsConfigDict(
+        env_file=".env",
+        env_file_encoding="utf-8",
+        extra="ignore",
+    )
+
+    # Server configuration
+    host: str = "0.0.0.0"
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)